invenio_app_rdm/utils/utils.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+# Copyright (C) 2024 KTH Royal Institute of Technology.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Utilities for building connection strings."""
+
+import os
+
+from click import secho
+
+
+def build_db_uri():
+    """Build the database URI."""
+    DEFAULT_URI = "postgresql+psycopg2://invenio-app-rdm:invenio-app-rdm@localhost/invenio-app-rdm"
```

Comment:
I'm way out of my area now, so sorry if I'm talking nonsense now, but should we not use the existing `INVENIO_SQLALCHEMY_DATABASE_URI` variable rather than constructing a static default string here? But as long as we don't break existing behavior with the old environment variables, I don't care. :)

Reply:
Ah, I see what you mean (I hope!) The `INVENIO_SQLALCHEMY_DATABASE_URI` variable you're referring to is the same as `SQLALCHEMY_DATABASE_URI` that we're already handling here. The `INVENIO_` prefix is intended to override any default or constructed value.
Let me know if that clears it up or I am talking about something else 😅